app/helpers/policies_helper.rb

```diff
@@ -55,6 +55,7 @@ def scap_content_selector(form)
       select_f form, :scap_content_id, scap_contents, :id, :title,
                { :include_blank => _("Choose existing SCAP Content") },
                { :label => _("SCAP Content"),
+                 :required => true,
```

Comment:
This doesn't seem to add the asterisk
![image](https://github.com/theforeman/foreman_openscap/assets/7326770/2dcdf08e-e81c-477f-a4d0-bf86c5270e0c)

Reply:
I confirm, I see * on name in Policy attributes but not here